packages/logger/README.md

```diff
@@ -40,10 +40,52 @@ npm install @sapphire/plugin-logger @sapphire/framework
 ---
 
 ## Usage
-
+This registers the necessary options and methods in the Sapphire client to be able to use the log plugin.
 ```typescript
+// Main bot file
 import '@sapphire/plugin-logger/register';
 ```
+To be able to use the Logger anywhere you first need to import the `containter` property from the `@Sapphire/framework` package.
+```typescript
+import { container } '@sapphire/plugin-logger';
+
+export class MyAwesomeService {
+	public printAwesomeLog() {
+		container.logger.log('log message');
+	}
+}
+```
+In the commands, the container is already injected into the extended class "command", therefore it is enough to use the method `this.container` to be able to access the Logger method.
+```typescript
+// ping command
+
+import { Command, CommandOptions, PieceContext } from '@sapphire/framework';
+import type { Message } from 'discord.js';
+
+export class PingCommand extends Command {
+	public constructor(context: PieceContext, options: CommandOptions) {
+		super(context, {
+			...options,
+			description: 'ping pong'
+		});
+	}
+
+	public async run(message: Message) {
+		this.container.logger.warn('warning message');
+	}
+}
+```
+
+## Types of logs
+1. debug
```

Comment:
1. trace
2. debug
3. info
4. warn
5. error
6. fatal

Remove write and adhere to this proper ascending order.

See https://github.com/sapphiredev/framework/blob/main/src/lib/utils/logger/Logger.ts#L14-L36